content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -89,29 +89,27 @@ the training process. To enable this, you will need to set up a
 Now we need to add a training stage to `dvc.yaml` including `checkpoint: true`
 in its <abbr>output</abbr>. This tells DVC which <abbr>cached</abbr> output(s)
 to use to resume the experiment later (a circular dependency). We'll do this
-with `dvc stage add`.
+with `dvc exp init`.
```

Comment:
Can this still be done with `dvc stage add`? Should we still mention it separately if so?

Reply:
In this specific case, it cannot be done with `stage add`.
In https://github.com/iterative/dvc/pull/8000 I have introduced some hidden options used by `exp init --type checkpoint` to make `metrics` and `plots` have `persist: true`.